data/preset/escola.yaml

```diff
@@ -17,22 +17,33 @@ defines:
       shortdef: 'Layout.Box(orientation: vertical)'
       slots:
         contents:
-        - shortdef: Layout.Box
+        - type: Layout.Overlay
           styles:
             - landing
           slots:
-            contents:
-            - shortdef: 'Banner.Dynamic(show-subtitle: true, mode: text, expand: true)'
-        - shortdef: 'ContentGroup.ContentGroup(expand: true)'
-          slots:
-            <<: *main-arrangement-slot
-            selection: Selection.AllSets
-        - shortdef: 'Layout.Box(orientation: vertical, halign: fill, hexpand: true)'
-          styles:
-            - footer
+            content:
+              shortdef: Layout.Box
+              styles:
+                - background
+            overlays:
+            - shortdef: 'Layout.Box(orientation: vertical, valign: center, vexpand: true)'
+              slots:
+                contents:
+                - shortdef: 'Banner.Dynamic(mode: text, expand: true)'
+                - shortdef: 'ContentGroup.StaticText(label: "TV ESCOLA")'
```

Comment:
I disagree. In fact it's quite likely we'll be asked to make apps for 3.1.x point releases that use this template, and I don't want to backport the parametrization stuff if we can help it. I think this is simply a place where we should compromise on the design.